PSKoans/Public/Measure-Karma.ps1

```diff
@@ -65,16 +71,24 @@ function Measure-Karma {
             Write-MeditationPrompt -Greeting
 
             Write-Verbose 'Sorting koans...'
+
             $SortedKoanList = Get-ChildItem "$env:PSKoans_Folder" -Recurse -Filter '*.Koans.ps1' |
                 Get-Command {$_.FullName} |
-                Where-Object {$_.ScriptBlock.Attributes.TypeID -match 'KoanAttribute'} |
-                Sort-Object {
-                $_.ScriptBlock.Attributes.Where( {$_.TypeID -match 'KoanAttribute'}).Position
-            }
-
+                ForEach-Object {
+                    $_Koan = $_
+                    $_.ScriptBlock.Attributes |
+                        Where-Object { $_.TypeID -Match 'KoanAttribute' } |
+                        ForEach-Object {
+                            $_.FilePath = $_Koan.Path;
+                            $_.FileCmdlet = $_Koan;
```

Comment:
These properties need better names. It's not at all clear to me what's going on here. :/

You're storing a reference to the `Get-Command`-retrieved object? I think? Maybe `CommandInfo` would be a better name?